App.js

```diff
@@ -1,25 +1,45 @@
-import React from "react";
+import React, { useState } from "react";
 import { NavigationContainer } from '@react-navigation/native';
 import { MainContainer } from "./src/components/Main/main.container";
 import { GameScreenContainer } from "./src/components/gameScreen/gameScreen.container";
 import { GameOverContainer } from "./src/components/gameOver/gameOver.container";
 import { createNativeStackNavigator } from "@react-navigation/native-stack";
 import { BookmarkScreenContainer } from "./src/components/bookmarkScreen/bookmarkScreen.container";
 import { QuestionsSetScreenContainer } from "./src/components/questionsSetScreen/questionsSetScreen.container";
+import { ApiProvider } from "./src/utils/api";
+import { BookmarksContext } from "./src/utils/bookmarks";
+import {ErrorBoundary} from 'react-error-boundary'
```

Comment:
you don't need a library, implement error boundary with built-in features of React